presto-parser/src/main/java/com/facebook/presto/sql/tree/ArithmeticUnaryExpression.java

```diff
@@ -38,12 +51,12 @@ public ArithmeticUnaryExpression(Sign sign, Expression value)
 
     public static ArithmeticUnaryExpression positive(Expression value)
     {
-        return new ArithmeticUnaryExpression(Sign.PLUS, value);
+        return new ArithmeticUnaryExpression(Optional.empty(), Sign.PLUS, value);
     }
 
     public static ArithmeticUnaryExpression negative(Expression value)
```

Comment:
I believe this need to have SqlLocation. It's a factory method called directly from `AstBuilder`